Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -1129,8 +1129,7 @@ void HTMLInputElement::computed_properties_changed()
     if (!appearance.has_value() || *appearance == CSS::Appearance::None)
         return;
 
-    auto palette = document().page().palette();
-    auto accent_color = palette.color(ColorRole::Accent).to_string();
+    auto accent_color = CSS::CSSKeywordValue::create(CSS::Keyword::Accentcolor)->to_string(Web::CSS::CSSStyleValue::SerializationMode::Normal);
```

Comment:
This is equivalent to `auto accent_color = "accentcolor";`, is that intentional? If so, the simpler way of doing that is `CSS::string_from_keyword(CSS::Keyword::Accentcolor)`.

Reply:
Done :+1: